web-console/4-patterns/notifications.md

```diff
@@ -0,0 +1,68 @@
+- A "New" notification is displayed with bold text because it is an unread notification. **Notifications are marked as read by clicking on the cell to unbold the text.**
+- **Users can clear notifications individually or by clicking the clear all button to remove all notifications from the currently open accordion section.**
+- There are two types of notifications that will be surfaced, System Alerts and Events.
+- An accordion style is used inside the drawer for opening and closing one section at a time.
+- If the drawer is closed and reopened, your last opened section will remain open.
+
+### System Alerts (OpenShift ONLINE Only)
+- For OpenShift Online, the first accordion section should always be the "System Alerts."
+- Previously System Alerts were shown as a warning icon directly in the masthead.
+- For any new System Alerts, badges will be shown until the message is marked as read.
+- **Anytime there are unread System Alerts, this section of the accordion should default to open when users open the notification drawer.**
+  - This would stray from PatternFly, but would call attention to System Alerts to be sure users saw them.
+- The "xx New" label will always be displayed for the System Alerts section.
+
+### Events
+- Each Project will receive its own accordion section below the System Alerts (if present).
+- Within a Project, the Events can be of two types, either Warnings or Informational.
+- Toast notifications will also show for the majority of events that occur.
+
+![Notifications](img/Catalog_Drawer-3.png)
+- When there are no Projects created, the notification drawer will be in the empty state
+- Note: for OpenShift Online, the System Alerts section would be displayed at the top.
+
+![Notifications](img/Catalog_Drawer-4.png)
+- When a Project is created, a new accordion section will be added to the top of the drawer (under System Alerts if they exist).
+- **When on the homepage, opening the Notification Drawer for the first time will open the section for the most recent Project created (unless new system alerts exist).**
+- **The label showing the number of new notifications will only be displayed for the Project section that is currently open.**
+
+  [IMAGE]
+- **There may be a delay while the notifications load for a specific project (a spinner icon should be displayed while waiting).**
+
+#### Inside the Console
+
+![Notifications](img/Console_Drawer-1.png)
+- **When first opening the drawer inside the console, the open accordion section should default to the current Project the user is working in.**
+- The badge will appear when the current Project, (in this case, Sample Project Name) has a New notification.
+- **Other Projects will not get updated when new notifications come in because their accordion section is not open (notice there is no label below “Project B”).**
+- Users can check other Projects for notifications by opening another section in the accordion.
```

Comment:
So we'll check for new alerts for that project when the user opens the accordion? I'm not sure that behavior is obvious, and we're not indicating that in any way. 

What if underneath the project name we said something like "Expand to check for new alerts" or something? I think that would make it clearer that we're not actively checking for new alerts in that project, but that the user can get those alerts without navigating to that project.